components/List.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+
+export const List = (props: { arr: any[] }) => {
+  return (
+    <div style={{ margin: '0.5rem' }}>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              {Object.keys(Object.assign({}, ...props.arr)).map((val, id) => (
```

Comment:
```
Object.keys(props.arr)
```

Reply:
If I use this "Object.keys(Object.assign({}, ...props.arr)).map((val, id)", then the names of the columns appear in the headers
But if I use this "Object.keys(props.arr).map((val, id)" then I get numbers in the names of the columns